articles/connections/database/custom-db/templates/change-password.md

```diff
@@ -353,45 +375,36 @@ function changePassword (email, newPassword, callback) {
 
 ```sql
 function changePassword(email, newPassword, callback) {
-
-  var oracledb = require('oracledb');
+  const bcrypt = require('bcrypt');
+  const oracledb = require('oracledb');
   oracledb.outFormat = oracledb.OBJECT;
 
   oracledb.getConnection({
       user: configuration.dbUser,
       password: configuration.dbUserPassword,
-      connectString: "CONNECTION_STRING" // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
+      connectString: 'CONNECTION_STRING' // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
     },
     function(err, connection) {
-      if (err) {
-        return callback(new Error(err));
-      }
+      if (err) callback(err);
+
       bcrypt.hash(newPassword, 10, function(err, hash) {
-        if (err) { return callback(err); }
-        connection.execute(
-          "update users set PASSWORD = :hash " +
-          "where EMAIL = :email", [hash, email], { autoCommit: true },
-          function(err, result) {
-            if (err) {
-              doRelease(connection);
-              return callback(new Error(err));
-            }
-            doRelease(connection);
-            callback(null, result.rowsAffected > 0);
-          });
+        if (err) return callback(err);
+
+        const query = 'update Users set PASSWORD = :hash where EMAIL = :email';
```

Comment:
In Heroku review app, code block renders strangely from here on (green, when should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/change-password#oracle